tests/configuration_spec.lua

```diff
@@ -54,6 +54,43 @@ describe("configuration", function()
         })
     end)
 
+    it("can define and use multi-byte mappings", function()
+        require("nvim-surround").setup({
+            surrounds = {
+                -- multi-byte linguistic quote
+                ["’"] = {
+                    add = { "’", "’" },
+                    delete = "^()().-()()$",
+                }
+            }
+        })
+        set_lines({ "hey! hello world" })
+        set_curpos({ 1, 7 })
+        vim.cmd("normal ysiw’")
+        check_lines({ "hey! ’hello’ world" })
+        vim.cmd("normal ds’")
+        check_lines({ "hey! hello world" })
+    end)
+
+    it("can define and use 'interpreted' multi-byte mappings", function()
+        require("nvim-surround").setup({
+            surrounds = {
+                -- interpreted multi-byte quote
+                ["<M-]>"] = {
+                    add = { "[[", "]]" },
+                    delete = "^(%[%[)().-(%]%])()$",
+                }
+            }
+        })
+        set_lines({ "hey! hello world" })
+        set_curpos({ 1, 7 })
+        -- note: `:execute` necessary to interpret `\<M-]>` as raw termcodes
```

Comment:
Do you mind reverting to `:normal` and just creating a local mapping using `vim.api.nvim_replace_termcodes`, similarly to how I've done in the first two lines of this file? For example,
```lua
local meta_close_bracket = vim.api.nvim_replace_termcodes(...)
vim.cmd("normal ysiw" .. meta_close_bracket)
```

Reply:
Yes that's actually what I ended up with, since I couldn't make it work locally... Well actually it still doesn't work but not sure why yet ^^'